### PR TITLE
Adding AJO Entity Tags in Schema

### PR DESCRIPTION
**Pull request by anshchauhan**
Adding AJO Entity Tags support for Journey/Campaigns

### CODE REVIEW

**Comment by anshchauhan** [comment]
@chetanmeh

File: extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json [hunk 1]
```diff
@@ -195,6 +195,19 @@
               "description": "Represents the journey node type as authored on the canvas"
             }
           }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/entities/tags": {
+          "title": "AJO Entity tags",
+          "type": "object",
+          "description": "AJO Entity tags corresponding to a Journey or Campaign",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/tagValues": {
```
Comment (chetanmeh) [inline extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json:204]: May be just call it `values`. Also add a sample json

**Comment by anshchauhan** [comment]
@mjasrotia @prabhum2 Could you please review?

**Review by mjasrotia — APPROVED** [review]

**Comment by anshchauhan** [comment]
@mjasrotia could you please merge it as well.

**Pull request merged**